pkg/genesis/vesting_accounts_test.go

```diff
@@ -118,6 +118,12 @@ func (s *VestingModuleTestSuite) TestGenesisDelayedVestingAccountsTransferUnLock
 	vAcc, name, err := s.getAnUnusedDelayedVestingAccount(0)
 	require.NoError(s.T(), err)
 
+	// Since we want a partially unlocked balance we need to wait until
+	// the endTime is passed if not already
```

Comment:
this comment doesn't make sense to me. If this test needs a partially unlocked balance, I would expect it to wait until the start time has passed.

If it actually needed to wait until end time passed, then I would expect the helper above to be:

```go
	vAcc, name, err := s.getAnUnusedDelayedVestingAccount(tmtime.Now().Unix())
```

```suggestion
	// Since we want a partially unlocked balance we need to wait until
	// the startTime has passed if not already
```